atd-vze/src/queries/people.js

```diff
@@ -16,6 +16,12 @@ export const GET_PEOPLE = gql`
       person_type {
         prsn_type_desc
       }
+      gender {
+        gndr_desc
```

Comment:
I found one small bug in the gender and race/ethnicity dropdown [default values](https://github.com/cityofaustin/atd-vz-data/blob/33a8a797190ab58f0436f601864b4b77932c0722/atd-vze/src/views/Crashes/RelatedRecordsTable.js#L142-L148). They just need `gndr_id` and `ethnicity_id` coming in from these person and primary person queries.

You can replicate by:
1. Click the pencil for gender or race/ethnicity of a person or primary person
2. See `UNKNOWN` as the select default value instead of the current value

Reply:
👍 
https://user-images.githubusercontent.com/14793120/204333682-5ccb04e6-76e5-4326-aefd-fb501a0752d4.mov

Reply:
Wow thanks for pointing this out Mike! Just pushed up this fix